hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -223,7 +222,8 @@ public static HoodieTableMetaClient createMetaClient(StorageConfiguration<?> sto
    */
   public static HoodieTableMetaClient createMetaClient(Configuration conf,
                                                        String basePath) {
-    return createMetaClient(HoodieStorageUtils.getStorageConfWithCopy(conf), basePath);
+    return createMetaClient((StorageConfiguration<Configuration>) ReflectionUtils.loadClass(HADOOP_STORAGE_CONF,
```

Comment:
Should this static method be moved to `hudi-hadoop-common` and directly use the constructor of `HadoopStorageConfiguration`?

Reply:
yeah